src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts

```diff
@@ -1709,6 +1726,64 @@ export class ConfigureRecommendedExtensionsCommandsContributor extends Disposabl
 			},
 			when: this.workspaceFolderContextKey
 		});
+
+		CommandsRegistry.registerCommand(AddToWorkspaceRecommendationsAction.ID, serviceAccesor => {
+			this.boundAddToRecommendationsContextKey.set(false);
+			const extensionId = (this.editorService.activeEditor as ExtensionsInput).extension.id;
+			serviceAccesor
+				.get(IInstantiationService)
```

Comment:
Style nit: Formatting here is pretty different from the above cases, which makes it less immediately obvious that they're all doing the same general thing.